test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -71,7 +71,7 @@ echo "presubmit test starts"
 
 # activating the service account
 gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
-gcloud config set compute/zone us-west1-a
+gcloud config set compute/zone us-east1-a
```

Comment:
east1-a is not a valid region. please switch to east1-b/c

Reply:
done